packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -521,6 +521,69 @@ const DatePickerInput = (props) => {
     }
   };
 
+
+
+  const handleInputKeydown = (event, inputType) => {
+    const { day, month, year } = date;
+    let inputDate;
+    let formattedDate;
+    if (day.length === 2 && month.length === 2 && year.length === 4) {
+      inputDate = DateUtil.convertToISO8601(`${year}-${month}-${day}`, DateUtil.ISO_EXTENDED_DATE_FORMAT);
+      formattedDate = DateUtil.strictFormatISODate(inputDate, momentDateFormat);
+    }
+    const validDate = DateUtil.isValidDate(formattedDate, momentDateFormat);
+
+    // set date to today
```

Comment:
should we restrict this shortcuts to work only for input type day 
`inputType === DateUtil.inputType.DAY` . Since we can only increment or decrement or set only a day field with this shortcuts. should we prevent increment and decrement of day when the focus is on month / year input when`T` / `t` or `-` or `+` keys are pressed.

other inputs on date-picker does already support arrow keys for increment and decrement of year, month and day when the focus is on respective input.

Reply:
Thanks @supreethmr, the current code is functioning as intended, mimicking how the legacy date and time inputs work in Cerner Win32/Win64 products, where the keyboard shortcut commands for `T`  `-` and `+`  work anywhere within the date field input (it visually looks like a single input as well), and will be what users expect.

Although I noticed it felt slightly odd during testing as well, that it appears much more visually noticeable in Terra than it does in the Win32/Win64, so I will be investigating further to see what micro-interaction differences there are, and what we can do to improve the visual continuity to new key listeners working equally in any node/input (month/day/year/etc) without calling specific noticeability to just the day (and minute) inputs.

Also for awareness, there will be more future stories where we add even more existing Cerner key shortcuts that will have additional behaviors that apply to the full date value ('W' for start of the week, 'K' for the end of the week, 'M' for the start of the month, 'H' for the end of the month, 'Y' for start of the year, 'R' for end of the year, and a few others), so they will have more effect than simply the Day and will also work in any of the inputs.